help-fns+.el

```diff
@@ -109,7 +109,7 @@
 ;;    (require 'help-fns+)
 ;;
 ;;  Acknowledgement: Passing text properties on doc strings to the
-;;  *Help* buffer is an idea from Johan bockgard.  He sent it on
+;;  *Help* buffer is an idea from Johan Bockgard.  He sent it on
```

Comment:
The name is actually `Johan Bockgård`, but I didn't want to try to enter that into the file (over ssh) -- I'm not sure if the file format would like the character...